src/components/Product.js

```diff
@@ -1,14 +1,12 @@
-import React, {Component} from 'react';
+const Product = (props) => {
```

Comment:
Another thing you can do to optimize what you get as props it's maybe parsing the data to just get the properties you are using. because as you can see for example in your image source you don't need everything inside item you just need the url and alt. Your product list in the end could be a very simple array of objects like 
```
[ { url, alt, name, slug, price }, { url, alt, name, slug, price } ]
```
its okay if you leave it as it is. Just think that it would be very helpful to pass from a data.property1.subproperty2 to just property. The section of code you have below can end like this
```
<div className="wz-col wz-mobile-full wz-thirds wz-product">
            <img src={url} alt={alt} />
            <h4 className="wz-center">{name}</h4>
            <p className="wz-center">{slug}</p>
            <p className="wz-center">$ {price}</p>
</div>
```
Also a lot of clients have this like part of their code convention so a PR sometimes can be rejected because this simple things.